buildtools/u2204/env-asan.inc

```diff
@@ -0,0 +1,19 @@
+source buildtools/u2004/env.inc
+export CFLAGS="\
+ -Wall\
+ -Werror\
+ -Wno-deprecated-declarations\
+ -Wno-unused-variable\
+ -Wno-unused-function\
+ -g3\
+ -ggdb3\
+ -fsanitize=address\
+ -fsanitize-recover=address"
+export CXXFLAGS="$CXXFLAGS -g3 -ggdb3 -fsanitize=address -fsanitize-recover=address"
```

Comment:
```suggestion
export CXXFLAGS="$CXXFLAGS -g3 -fsanitize=address -fsanitize-recover=address"
```

Reply:
not part of this PR (was divergent commits, have been resolved)